README.rst

```diff
@@ -55,6 +55,37 @@ Dev mode
 
    pip install -e .[dev]
 
+Version control of database using DVC (Data Version Control)
+------------
+
+Data Version Control (DVC) is a data management tool that is meant to be run alongside Git.
+In this project DVC is used to link changes in the code to specific versions of the database. DVC can be
```

Comment:
It's not clear to the reader what is "the database".
Maybe we can call it "database folder of example data" or something like that.

or be more generic and just call it "project data such as an example database" so that it holds true no matter what kind of project-associated data we end up trackign with dvc

Reply:
And thanks for sharing the RST tool!